src/cli/main.ts

```diff
@@ -0,0 +1,44 @@
+import yargs from 'yargs/yargs';
+import {Worker} from 'worker_threads';
+import {CommandProcessor} from './command-processor';
+
+const APP_INSTANCE_PATH = './build/src/cli/app-instance.js';
+
+/**
+ * Process initial command-line arguments.
+ */
+const argv = yargs()
+  .usage('Usage $0 --app_path PATH')
```

Comment:
Can we simplify the command-line arg to a single word to simplify the call? ie. `--app /path/to/app`.

Reply:
Done